src/main/frontend/src/components/aggregates/AggregateDialog.svelte

```diff
@@ -18,94 +12,83 @@
 
 	export let dialogActive;
 	export let editMode;
+	export let oldAggregate;
+	let newAggregate;
+	let aggregateName, stateFields, events, methods, rootPath, producerExchangeName, consumerExchangeName, schemaGroup, disableSchemaGroup, routes, outgoingEvents, receivers;
```

Comment:
less to rewrite, as we now have one `initialAggregate` do define init/clear-state

Reply:
looks good!